ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/components/ExceptionRequestTableCells.tsx

```diff
@@ -155,3 +167,49 @@ export function RequestedItems({ exception, context }: RequestedItemsProps) {
     }
     return <div>{`${cvesCount} ${pluralize('CVE', cvesCount)}`}</div>;
 }
+
+export type RequestCommentProps = {
+    comment: VulnerabilityExceptionComment;
+};
+
+export function RequestComment({ comment }: RequestCommentProps) {
+    return (
+        <Flex direction={{ default: 'column' }}>
+            <Flex direction={{ default: 'row' }} spaceItems={{ default: 'spaceItemsSm' }}>
+                <Text className="pf-u-font-weight-bold" component={TextVariants.p}>
+                    {comment.user.name}
+                </Text>
+                <Text component={TextVariants.small}>({getDateTime(comment.createdAt)})</Text>
+            </Flex>
+            <FlexItem>{comment.message}</FlexItem>
+        </Flex>
+    );
+}
+
+export type RequestCommentsProps = {
+    comments: VulnerabilityExceptionComment[];
+};
+
+export function RequestComments({ comments }: RequestCommentsProps) {
+    const { isModalOpen, openModal, closeModal } = useModal();
+
+    return (
+        <>
+            <Button variant="link" isInline onClick={openModal}>{`${comments.length} ${pluralize(
+                'comment',
+                comments.length
+            )}`}</Button>
+            <Modal variant="small" title="Comments" isOpen={isModalOpen} onClose={closeModal}>
+                <List isPlain isBordered>
+                    {comments.map((comment) => {
+                        return (
+                            <ListItem>
```

Comment:
```suggestion
                            <ListItem key={comment.id}>
```

Reply:
Ah, I keep forgetting about this one 🤦🏼‍♂️